From 2c7c4a9c23950044507a78956ca2c23f9c6a9491 Mon Sep 17 00:00:00 2001
From: Jeremy Harris <jgh146exb@wizmail.org>
Date: Thu, 14 Feb 2019 17:14:34 +0000
Subject: [PATCH 05/29] Fix info on using local_scan() in the default Makefile

Broken-by: 9723f96673
(cherry picked from commit 882bc1704d33aa34873e3a0f72e657b0cc2985e5)
(cherry picked from commit cb25b75af850d664fc005d24fbad0e58bf79d4c7)
---
 OS/Makefile-Default | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/OS/Makefile-Default b/OS/Makefile-Default
index b3990fe8..41a4dbbd 100644
--- a/OS/Makefile-Default
+++ b/OS/Makefile-Default
@@ -232,6 +232,11 @@ RANLIB=ranlib
 EXIM_CHMOD=@true
 
 
+# If you want to use local_scan() at all, the support code must be included
+# by uncommenting this line.
+
+# HAVE_LOCAL_SCAN=yes
+
 # LOCAL_SCAN_SOURCE defines the file in which the function local_scan() is
 # defined. This provides the administrator with a hook for including C code
 # for scanning incoming mails. The path that is defined must be relative to
@@ -239,8 +244,9 @@ EXIM_CHMOD=@true
 
 # LOCAL_SCAN_SOURCE=Local/local_scan.c
 
-# The default setting points to a template function that doesn't actually do
-# any scanning, but just accepts the message.
+# A very simple example points to a template function that doesn't actually do
+# any scanning, but just accepts the message.  A compilable file must be
+# included in the build even if HAVE_LOCAL_SCAN is not defined.
 
 LOCAL_SCAN_SOURCE=src/local_scan.c
 
-- 
2.20.1

